pretty_cron/api.py

```diff
@@ -21,7 +25,7 @@ def prettify_cron(expression):
                 # non-integers in comma-separated list aren't supported yet -
                 # return as-is
                 return expression
-        elif piece != '*':
+        elif piece != '*' and not (step_pattern.match(piece) and i in (0, 1)):
```

Comment:
It's a shame the regex is being parsed twice on each piece. I wonder if this code can be refactored to do the match once and then have `_pretty_time_with_steps` pick up the parts of the match from `pieces`